packages/forms/src/widgets/CodeWidget/CodeWidget.component.js

```diff
@@ -1,50 +1,53 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import TextareaWidget from 'react-jsonschema-form/lib/components/widgets/TextareaWidget';
```

Comment:
Note this is the OLD widget from mozilla form libs which is not anymore used and will be removed in 3.0